packages/web-components/src/components/masthead/megamenu-category-link.ts

```diff
@@ -59,13 +59,13 @@ class C4DMegaMenuCategoryLink extends CDSLink {
     return html`
       ${title
         ? html`
-            <div part="link-heading">
```

Comment:
I'd rather leave this as is, in case it's already being used in the wild. This is an API break.